Resources/Prototypes/Maps/pebble.yml

```diff
@@ -12,7 +12,7 @@
           mapNameTemplate: '{0} Pebble Station {1}'
           nameGenerator:
             !type:NanotrasenNameGenerator
-            prefixCreator: 'NY'
+            prefixCreator: 'DV'
```

Comment:
Don't change this. I want to keep the legacy of the server it was originally created on.